src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -118,6 +124,30 @@ def _list_devices(self, cmd):
 
         return 0, result, ""
 
+    def _list_services(self, cmd):
+        svc_type = cmd.get('svc_type', None)
+
+        completion = self._oremote("list_services", svc_type)
+        self._wait([completion])
+        services = completion.result
+
+        if len(services) == 0:
+            return 0, "", "No services reported"
+        else:
+            # Sort the list for display
+            services.sort(key = lambda s: s.daemon_name)
+            services.sort(key = lambda s: s.nodename)
+            services.sort(key = lambda s: s.service_type)
```

Comment:
You can sort with a tuple as the key:
```suggestion
            services.sort(key = lambda s: (s.service_type, s.nodename, s.daemon_name)))
```

Reply:
Thanks, I'll change that.